src/parser/jobs/pct/modules/Procs.tsx

```diff
@@ -0,0 +1,87 @@
+			consumeActions: [
+				this.data.actions.AERO_IN_GREEN,
+				this.data.actions.AERO_II_IN_GREEN,
+				this.data.actions.STONE_IN_YELLOW,
+				this.data.actions.STONE_II_IN_YELLOW,
+			],
+		},
+		{
+			procStatus: this.data.statuses.AETHERHUES_II,
+			consumeActions: [
+				this.data.actions.WATER_IN_BLUE,
+				this.data.actions.WATER_II_IN_BLUE,
+				this.data.actions.THUNDER_IN_MAGENTA,
+				this.data.actions.THUNDER_II_IN_MAGENTA,
+			],
+		},
+		{
+			procStatus: this.data.statuses.RAINBOW_BRIGHT,
+			consumeActions: [
+				this.data.actions.RAINBOW_DRIP,
+			],
+		},
+		{
+			procStatus: this.data.statuses.STARSTRUCK,
+			consumeActions: [
+				this.data.actions.STAR_PRISM,
+			],
+		},
+	]
+
+	protected override jobSpecificOnConsumeProc(procGroup: ProcGroup, event: Events['action']): void {
+		// Rainbow bright changes Rainbow Drip from a 4s cast/6s recast to a standard instant GCD
+		if (procGroup.procStatus.id === this.data.statuses.RAINBOW_BRIGHT.id) {
+			this.castTime.setInstantCastAdjustment([this.data.actions.RAINBOW_DRIP.id], event.timestamp, event.timestamp)
+			this.castTime.setRecastTimeAdjustment([this.data.actions.RAINBOW_DRIP.id], RAINBOW_BRIGHT_RECAST_ADJUSTMENT)
+		}
+	}
+
+	protected override addJobSpecificSuggestions(): void {
+		const droppedAetherhues = this.getDropCountForStatus(this.data.statuses.AETHERHUES.id) + this.getDropCountForStatus(this.data.statuses.AETHERHUES_II.id)
+		const droppedRainbows = this.getDropCountForStatus(this.data.statuses.RAINBOW_BRIGHT.id)
+		const droppedStars = this.getDropCountForStatus(this.data.statuses.STARSTRUCK.id)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.statuses.AETHERHUES_II.icon,
+			content: <Trans id="pct.procs.dropped-aetherhues.content">Your <DataLink status="AETHERHUES" /> and <DataLink status="AETHERHUES_II" /> statuses function as Pictomancer's combos, and dropping them will slow the rate at which you're able to use <DataLink action="SUBTRACTIVE_PALLETTE" />, leading to lost potency over time.</Trans>,
+			why: <Trans id="pct.procs.dropped-aetherhues.why">One of your <DataLink showIcon={false} status="AETHERHUES" /> statuses were allowed to expire <Plural value={droppedAetherhues} one="# time" other="# times"/>.</Trans>,
+			value: droppedAetherhues,
+			tiers: DEFAULT_SEVERITY_TIERS,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.statuses.RAINBOW_BRIGHT.icon,
+			content: <Trans id="pct.procs.dropped-rainbows.content"><DataLink action="RAINBOW_DRIP" /> is one of your strongest GCDs, and <DataLink status="RAINBOW_BRIGHT" /> allows its use as a normal-length instant GCD, rather than requiring an extremely long cast time.</Trans>,
+			why: <Trans id="pct.procs.dropped-rainbows.why"><DataLink showIcon={false} status="RAINBOW_BRIGHT" /> was allowed to expire <Plural value={droppedRainbows} one="# time" other="# times" />.</Trans>,
+			value: droppedRainbows,
+			tiers: DEFAULT_SEVERITY_TIERS,
```

Comment:
_Consider:_ Again, just an armchair perspective, but given the importance of Rainbow Drip and Star Prism, it might be worth making the tiers stricter for both this suggestion and the one for Starstruck below.

Reply:
Fixed now, had meant to do 1: medium, 2: major and just... forgot